deployments/config.json

```diff
@@ -4,15 +4,17 @@
             "selectors": {
                 "vendors": ["8086"],
                 "devices": ["154c", "10ed"],
-                "drivers": ["i40evf", "ixgbevf"]
+                "drivers": ["i40evf", "ixgbevf"],
+                "ifnames": ["ens0", "enp2s0f1"]
```

Comment:
Lets keep the sample deployment configs simple and more general so that user can run deploy them easily. For advanced use-cases proper documentation will be fine I'd say.

Reply:
"pfNames" is removed from the first example